docs/docsite/rst/user_guide/collections_using.rst

```diff
@@ -273,7 +273,9 @@ Verifying against ``tar.gz`` files is not supported. If your ``requirements.yml`
 Signature verification
 ----------------------
 
-If a collection has been signed by the Galaxy server, the server will provide ASCII armored, detached signatures to verify the authenticity of the MANIFEST.json before using it to verify the collection's contents. You must opt into signature verification by :ref:`configuring a keyring <galaxy_gpg_keyring>` for ``ansible-galaxy``, or by providing the path with the ``--keyring`` option.
+If a collection has been signed by the Automation Hub or self-hosted pulp or on-premise Hub server, the server will provide ASCII armored, detached signatures to verify the authenticity of the MANIFEST.json before using it to verify the collection's contents. This option is not available on community Galaxy.
```

Comment:
I'm assuming "Automation Hub, self-hosted pulp, and on-premise hub server" are all different servers that provide ASCII armored?

Still don't understand the difference between the Galaxy server and those mentioned above.

Reply:
`Galaxy` refers to galaxy.ansible.com.

That's not what was meant here, hence the fix, but the docs use `Galaxy server` often even when it may mean/include AH/pulp, perhaps since we have configuration options called `GALAXY_SERVER` and `GALAXY_SERVER_LIST` (which are used to configure Ansible with any galaxy server, not just Galaxy).

Reply:
Yes thanks to both of you. This requires a bit more thought. We have :

- Galaxy - which historically meant galaxy.ansible.com
- Automation Hub,  -which is a Red Hat product
- selfhosted (which can be Automation Hub, and I think what the repository calls galaxyNG )

I don't actually know what pulp is, outside of galaxyNG self-hosted?  But I think what I need to do is start a conversation on naming, because we will definitely have an overalp of galaxy.ansible.com and galaxyNG to deal with. I'll put this PR in WIP for now.

Reply:
Looks like we've resolve this to:

- Galaxy - refers to galaxy.ansible.com
- Self-hosted Galaxy refers to the GalaxyNG codebase

I'll update the PR and add this to the glossary as well.